src/components/contribute/index.js

```diff
@@ -479,6 +488,17 @@ export class Contribute extends React.Component {
       'qr-selected': contributeThrough === CONTRIBUTION_OPTIONS.QR
     })
 
+    const crowdsaleAddress = (crowdsale && crowdsale.execID) || (MintedCappedProxy && MintedCappedProxy.addr)
+    const crowdsaleAddressTruncated =
+      (crowdsale && truncateStringInTheMiddle(crowdsale.execID)) ||
+      (MintedCappedProxy && truncateStringInTheMiddle(MintedCappedProxy.addr))
+    const crowdsaleAddressDescription = crowdsale
+      ? crowdsale.execID
+        ? 'Crowdsale Execution ID'
+        : 'Crowdsale Proxy Address'
+      : 'Crowdsale ID'
+    const crowdsaleAddressTooltip = `Crowdsale execution ID to copy: ${crowdsaleAddress}`
```

Comment:
@mariano-aguero for execution ID the tooltip text should be `Crowdsale execution ID to copy:`. But for proxy address - `Crowdsale proxy address to copy:`

Reply:
Hi @vbaranov , changes made. Thanks for the feedback